crates/stages/src/metrics/listener.rs

```diff
@@ -44,10 +49,14 @@ impl MetricsListener {
 
     fn handle_event(&mut self, event: MetricEvent) {
         match event {
+            MetricEvent::SyncHeight { height } => {
+                for stage_id in StageId::ALL {
+                    let stage_metrics = self.sync_metrics.get_stage_metrics(stage_id);
+                    stage_metrics.checkpoint.set(height as f64);
+                }
+            }
```

Comment:
i see, nice - i guess we dont need to do anyhting with the entities processed / entities total?